scripts/dev-setup.sh

```
@@ -0,0 +1,114 @@
+    echo "Starting sippy on http://localhost:$SIPPY_API_PORT ..."
+    echo "Press Ctrl-C to stop"
+    echo ""
+
+    ./sippy serve \
+      --listen ":$SIPPY_API_PORT" \
+      --listen-metrics ":12112" \
+      --database-dsn="$DSN" \
+      --enable-write-endpoints \
+      --log-level debug \
+      --views config/e2e-views.yaml \
+      $GCS_ARGS \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)